theSrc/scripts/PlotData.coffee

```diff
@@ -23,19 +23,17 @@ class PlotData
     @origY = @Y.slice(0)
     @normX = @X.slice(0)
     @normY = @Y.slice(0)
-    @normZ = @Z.slice() if Utils.get().isArrOfNums(@Z) and @Z.length == @X.length
+    @normZ = @Z.slice() if Utils.isArrOfNums(@Z) and @Z.length == @X.length
     @outsidePlotPtsId = []
     @legendPts = []
     @outsidePlotCondensedPts = []
     @legendBubbles = []
     @legendBubblesLab = []
 
```

Comment:
cIndex is no longer needed and is not used anywhere (except old PlotColors code)